tests/MongoDB.Driver.Tests/MongoIndexManagerTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using FluentAssertions;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.XunitExtensions;
+using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.Operations;
+using Xunit;
+
+namespace MongoDB.Driver.Tests
+{
+    public class MongoIndexManagerTests
+    {
+        [Theory]
+        [ParameterAttributeData]
+        public void List_should_work_as_expected(
+            [Values("{ singleIndex : 1 }", "{ compoundIndex1 : 1, compoundIndex2 : 1 }")] string key,
+            [Values(false, true)] bool uniqueArgument,
```

Comment:
Rename to `unique` (see comment below that avoids name conflict).

Reply:
done